uml/sequence_diagrams/Power.txt

```diff
@@ -0,0 +1,86 @@
+@startuml
+participant Acceletometer
+participant PowerMonitor
+participant Battery
+participant BlueTooth
+participant Brightness
+participant HealtStatusCheck
+participant Storage
+participant VibrationMotor
+
+PowerMonitor --> PowerMonitor: PowerDown == ON
+loop event
+Alt event = AskBatteryEvent
+PowerMonitor --> Battery: AskBattery
+Activate Battery
+Battery --> PowerMonitor: BatterySoC
+Deactivate Battery
+alt BatterySoC<=10% AND BatterySoC>2% AND (LowPower == OFF OR PowerDown == ON)
+  PowerMonitor--> Brightness: change Brightness to value 25%
+  PowerMonitor--> BlueTooth: Disable BlueTooth
+  PowerMonitor--> HealtStatusCheck: Disable HealtStatusCheck
+  PowerMonitor--> VibrationMotor: Send vibration
+  PowerMonitor --> PowerMonitor: LowPower == ON
+  PowerMonitor --> PowerMonitor: PowerDown == OFF
+else BatterySoC>10% AND (LowPower == ON OR PowerDown == ON)
+  PowerMonitor--> Storage: readStorage(Brightness, BlueTooth, HealtStatusCheck)
+  Activate Storage
+  Storage--> PowerMonitor: Brightness, BlueTooth, HealtStatusCheck
+  Deactivate Storage
+alt Brightness != StorageError
+  PowerMonitor--> Brightness: change Brightness to value 'Brightness(Storage )'
+else
+  PowerMonitor--> Brightness: change Brightness to value 50%
+end 
+alt BlueTooth != StorageError
+  PowerMonitor--> BlueTooth: change BlueTooth to value 'BlueTooth(Storage )'
+else
+  PowerMonitor--> BlueTooth: change BlueTooth to ON
+end 
+alt HealtStatusCheck != StorageError
+  PowerMonitor--> HealtStatusCheck: change HealtStatusCheck to value 'HealtStatusCheck(Storage )'
+else
+  PowerMonitor--> HealtStatusCheck: change HealtStatusCheck to OFF
+end 
+  PowerMonitor --> PowerMonitor: LowPower == OFF
+  PowerMonitor --> PowerMonitor: PowerDown == OFF
+else BatterySoC<=2% AND PowerDown == OFF
+  PowerMonitor --> PowerMonitor: Deactivate Watch

```

Comment:
`event = manual deactivation`  to trigger a `Deactivate Watch` action could be another thing to consider (i.e. by user holding buttons)

Reply:
In this diagram there is no interaction with the user. It's a background activity.